cycles_for.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>cycles for</title>
+  <script>
+    for (var i = 10, j=0; i >= 0; i--,j++) {
+      if (i%2==1){
+        document.write(i+","+j+"<br>");
+      }
+    }
+    document.write("Value i at the end of the cycle: "+i+"<br>");
```

Comment:
Combine this with what you learned inthe previous example, and add a password checker that uses this syntax.